### ISSUE TITLE
Fix exploding training and validation loss after ~2500 epochs

### ISSUE THREAD

**Issue by mradovic38** [mradovic38/pixel-art-character-generation#4]
The loss appears to be exploding in the later phases of U-Net training.
